src/internals/lincowcell_async/mod.rs

```diff
@@ -384,10 +383,22 @@ mod tests {
         let cc = Arc::new(LinCowCell::new(data));
 
         let _ = tokio::join!(
-            tokio::task::spawn(rt_writer(cc.clone())),
-            tokio::task::spawn(rt_writer(cc.clone())),
-            tokio::task::spawn(rt_writer(cc.clone())),
-            tokio::task::spawn(rt_writer(cc.clone())),
+            tokio::task::spawn_blocking({
+                let cc = cc.clone();
+                move || rt_writer(cc)
+            }),
+            tokio::task::spawn_blocking({
+                let cc = cc.clone();
+                move || rt_writer(cc)
+            }),
+            tokio::task::spawn_blocking({
+                let cc = cc.clone();
+                move || rt_writer(cc)
+            }),
+            tokio::task::spawn_blocking({
+                let cc = cc.clone();
+                move || rt_writer(cc)
+            }),
```

Comment:
I am really unsure about this one: On the one hand, any busy-looping task spawned on the single-threaded executor used by `tokio::test` will starve the read and write transactions. But on the other hand, having a read transaction become a non-yielding operation might be a bit of a foot gun, so it might need at least a bit of extra documentation warning of this?

Paying the cost of an async mutex for a read-side that never actually requires it seems wasteful though. So maybe if this was done, the docs/changelog should just warn to check if read sides became blocking code due to this change? And suggest calling `yield_now` if necessary to avoid starving write transactions?

Maybe this was all discussed before and I am just not aware of that discussion?

Reply:
(But do note that this is not the read transaction starving the write transaction. Starvation happens on the task level, i.e. `write` is never even entered because the `rt_writer` tasks never yield and the `mt_writer` never actually run. Which is why I wrote that any busy loop will do this, it does not have to call `read` at all.)

Reply:
Yeah, I think making this spawn blocking is the right move here since they are now blocking.